### ISSUE TITLE
Include the "Fold a wall of MS" and "Create bend on solids" tutorials

### ISSUE THREAD

**Issue by ceremcem** [shaise/FreeCAD_SheetMetal#135]
"Fold a wall of Metal Sheet" and "Create bend on solids" tutorials should be included or linked in the README of this repo. As an example, I'm struggling to find my way out to use them. 

* Question/answer form of "Fold a wall of MS": https://forum.freecadweb.org/viewtopic.php?t=45519#p389955 (does not work for me)

**Comment by shaise** [comment]
Agree. @jaisekjames , do you have any tutorials on these functions?

**Comment by jaisekjames** [comment]
First draw a line in sketch [ length more than face ] on face need to bend.
![Fold ex 1](https://user-images.githubusercontent.com/44542605/102972967-f87b4e80-4521-11eb-9375-224c4075786e.png)

Then select face & sketch together, click on folding icon [ due to some issue, green face selection is not capturing in my screenshot.]
![Fold ex 3](https://user-images.githubusercontent.com/44542605/102972984-0204b680-4522-11eb-8bd1-b3da143dc8c5.png)

It will be folded.
![Fold ex 4](https://user-images.githubusercontent.com/44542605/102972999-09c45b00-4522-11eb-9da9-bcc347d3f853.png)

In case in two line in sketch[co-linear], it will create fold only where line is resent
![Fold ex 5](https://user-images.githubusercontent.com/44542605/102973037-1779e080-4522-11eb-9d54-a92d96f9ddb5.png)

Folded view
![Fold ex 6](https://user-images.githubusercontent.com/44542605/102973063-206ab200-4522-11eb-800f-350ea1ee5349.png)

**Comment by luzpaz** [comment]
bump

**Comment by luzpaz** [comment]
Anyone game to add this to the README?
